drivers/wdm/audio/drivers/ac97/common.h

```diff
@@ -137,7 +137,7 @@ class CAC97AdapterCommon : public IAC97AdapterCommon,
     NTSTATUS SetAC97Default (void);
 
     //
-    // Aquires the semaphore for AC97 register access.
+    // Acquires the semaphore for AC97 register access.
```

Comment:
Third party (somewhere from https://github.com/microsoftarchive/msdn-code-gallery-microsoft according to the commit log... so maybe we can consider this a fork?)